docs/source/crs/index.rst

```diff
@@ -5,8 +5,9 @@ The :class:`cartopy.crs.CRS` class is the very core of cartopy, all coordinate r
 in cartopy have :class:`~cartopy.crs.CRS` as a parent class, meaning all projections have
 the interface described below.
 
-.. autoclass:: cartopy.crs.CRS(proj4_params, globe=None)
+.. autoclass:: cartopy.crs.CRS()
```

Comment:
See https://github.com/SciTools/cartopy/issues/923#issuecomment-355062776 for why I did this